src/util/edit.cc

```diff
@@ -109,6 +109,23 @@ size_t EditEngine::undo() {
 }
 
 void EditEngine::applyChanges() {
+  for (auto it = address_mapping_.cbegin(); it != address_mapping_.cend();
```

Comment:
Add a comment what this loop does.

Reply:
I'll comment the code after implementing insert and delete.